docs/testing.rst

```diff
@@ -50,129 +50,3 @@ Create your repository locally and upload that.
     $ scp -r * <fedora_username>:fedorapeople.org:/project/anaconda/repos/anaconda_fixup_repo
 
 After the above is done, everything should be fixed and Packit should work again.
-
```

Comment:
The information in this file is only removed (it does not exist in the new repo). I don't know if it's entirely invalid now, just taking note so it can be found later if needed.

Reply:
You 're right. I will create a Jira for followups and include this one.